channels/printer/client/win/printer_win.c

```diff
@@ -408,6 +408,12 @@ static void printer_win_release_ref_driver(rdpPrinterDriver* driver)
 	rdpWinPrinterDriver* win = (rdpWinPrinterDriver*)driver;
 	if (win->references <= 1)
 	{
+		if (win->defaultPrinter)
```

Comment:
no need to check for `win->defaultPrinter != NULL` as `free(NULL)` is safe (and does the test internally)

Reply:
Done